appveyor.yml

```diff
@@ -5,10 +5,11 @@ environment:
     # https://github.com/FormidableLabs/converter-react/issues/34
     ROWDY_SETTINGS: "local.firefox"
   matrix:
-    - nodejs_version: 0.10
     - nodejs_version: 0.12
+    - nodejs_version: 4.stable
```

Comment:
Is this an actual version? (Sorry if I led you astray, I was giving pseudocode in earlier comment)